lua/orgmode/config/init.lua

```diff
@@ -34,6 +34,7 @@ function Config:extend(opts)
   opts = opts or {}
   self:_deprecation_notify(opts)
   self.opts = vim.tbl_deep_extend('force', self.opts, opts)
+  utils.keymap('n', '<Plug>(orgmode)', self.opts.mappings.prefix)
```

Comment:
Do we need this for something? I don't think it's useful by itself.

Reply:
whoops, that's a relic from earlier experimentation. Sorry